app/(main)/league/[leagueId]/entry/[entryId]/week/Week.test.tsx

```diff
@@ -224,15 +320,31 @@ describe('Week', () => {
   });
 
   test('should redirect back to entry page after successfully selecting a team', async () => {
+    mockUseAuthContext.isSignedIn = true;
+    (getCurrentUserEntries as jest.Mock).mockResolvedValue([
+      {
+        $id: '123',
+        name: 'Entry 1',
+        user: '123',
+        league: '123',
+        selectedTeams: [],
+        eliminated: false,
+      },
+    ]);
+
     render(
       <Week entry={entry} league={league} NFLTeams={NFLTeams} week={week} />,
     );
 
     const teamRadios = await screen.findAllByTestId('team-radio');
     fireEvent.click(teamRadios[0]);
 
+    await waitFor(() => {
+      expect(screen.getByTestId('weekly-picks')).toBeInTheDocument();
+    });
+
     await waitFor(() => {
       expect(mockPush).toHaveBeenCalledWith(`/league/${league}/entry/all`);
-    })
```

Comment:
Wouldn't we want to wait for the `/league/${league}/entry/all` url to be called before checking if `weekly-picks` is in the document?

Reply:
nope ... cause you are in the weekly picks page.  once the user selects a pick in a specific entry they should be routed back to the all entries page the user has for their league.